packages/volto/src/components/manage/Controlpanels/Controlpanel.test.jsx

```diff
@@ -45,4 +52,28 @@ describe('Controlpanel', () => {
 
     expect(container).toMatchSnapshot();
   });
+  it('renders a controlpanel component with error', async () => {
+    const { container, rerender } = render(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={['/controlpanel/date-and-time']}>
+          <Route path={'/controlpanel/:id'} component={Controlpanel} />
+          <div id="toolbar"></div>
+        </MemoryRouter>
+      </Provider>,
+    );
+    store.getState().controlpanels.update.loading = true;
+    store.getState().controlpanels.update.error.response.body.message =
+      "[{'message': 'Twitter username should not include the \"@\" prefix character.', 'field': 'twitter_username', 'error': 'ValidationError'}]";
```

Comment:
a string?

Reply:
Yes, this is a key of json returned by restapi which is a string.